rurusetto/wiki/templates/wiki/create_ruleset.html

```diff
@@ -48,6 +48,16 @@ <h1>Icon & Logo</h1>
                     </div>
                   </div>
             </div>
+            <div class="fieldWrapper">
+              <div class="mb-3 row">
+                 <label class="col-sm-2 col-form-label form-label">{{ form.dark_icon.label_tag }}</label>
```

Comment:
```suggestion
                 <label class="col-sm-2 col-form-label form-label">{{ form.light_icon.label_tag }}</label>
```
Because the website's default theme is dark so it need to change.